presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveTransactionalTable.java

```diff
@@ -218,11 +227,13 @@ private static void tryCompactingTable(CompactionMode compactMode, String tableN
                     .findFirst();
 
             if (startedCompaction.isEmpty()) {
+                log.warn("Compaction has not started yet");
```

Comment:
nit: i think logging at INFO is enough for that to show in CI logs. Higher severity is kinda not in-line with what testng does as even test-failure is logged as INFO.

Reply:
Also, how many log lines do we produce per test run?
How many log lines per this test class?

is there a risk of bloating the test output?
(this could make viewing test logs in the browser harder)

Reply:
@findepi not much, couple of extra lines per test run

Reply:
I'll remove that log line. Does not bring much of a value